api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/ModelLoader.kt

```diff
@@ -398,7 +398,7 @@ private fun createActivationLayer(config: LayerConfig, name: String): Layer {
 
 private fun createReLULayer(config: LayerConfig, name: String): Layer {
     return ReLU(
-        maxValue = config.max_value!!.toFloat(),
+        maxValue = config.max_value?.toFloat(),
```

Comment:
shouldn't it be left as it was as here the config is supposed to have this value

Reply:
I'm not fully understand notation, maxValue is nullable, whereas other fields are not.

Reply:
I was thinking that it shouldn't be nullable in config (and then the exception would be expected on !! as it was) but now I see that it might be configured as null and it has a special meaning thought. Btw @zaleslaw  is an author of ReLU so he would be the best  source of answer here how should be a nullable value interpreted from config